README.md

```diff
@@ -684,6 +691,31 @@ following context is generated to use for templating:
 }
 ```
 
+##### Footers
+
+A conventional commit's body may end with any number of structured key-value
+pairs known as [_footers_]. These consist of a string token naming the footer, a
+separator (which is either `: ` or ` #`), and a value, similar to [the git
+trailers convention][trailers]. For example:
+
+- `Signed-off-by: User Name <user.email@example.com>`
+- `Reviewed-by: User Name <user.email@example.com>`
+- `Fixes #1234`
+- `BREAKING CHANGE: breaking change description`
+
+When a conventional commit contains footers, the footers are passed to the
+template in a `footers` array in the commit object. Each footer is represented
+by an object with the following fields:
+
+- `"token"`, the name of the footer (preceeding the separator character)
+- `separator`, the footer's separator string (either `: ` or ` #`)
+- `value`, the value following the separator character
+- `breaking`, which is `true` if this is a `BREAKING CHANGE:` footer, and
+  `false` otherwise
+
+[_footers_]: https://www.conventionalcommits.org/en/v1.0.0/#specification
+[trailers]: https://git-scm.com/docs/git-interpret-trailers)
```

Comment:
This doesn't seem to be rendered correctly due to trailing ")" character. Also, can you use in-place links (`[alt](url)`) instead to make it aligned with the rest of README.md?